### PR TITLE
Test on docker meta

### PR DESCRIPTION
**Pull request by SpecialAro**
The tag number was not being parsed to the step given we didn't trigger a pull tag event. Will this work as it is @vraravam ?

### CODE REVIEW

**Review by vraravam — CHANGES_REQUESTED** [review]

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -49,6 +49,7 @@ jobs:
             git push origin main --no-verify
             git tag -f $TAG_NAME
             git push origin --tags --no-verify
+            export tag_name="${tag_name:1}"
```
[the remark this answers is not in the document]
Comment (SpecialAro) [reply .github/workflows/docker.yml]: Thank you @vraravam 😄 Pushed a new commit to solve this. Can you take a look, please?

File: .github/workflows/docker.yml [hunk 2]
```diff
@@ -71,7 +73,7 @@ jobs:
           # generate Docker tags based on the following events/attributes
           tags: |
             latest
-            type=semver,pattern={{version}}
+            type=raw,value=$tag_name
```
Comment (vraravam) [inline .github/workflows/docker.yml:76]: this line also needs to change to 
```
type=raw,value=${{ env.tag_name }}
```

Also, usually, all env vars are typically all caps with underscores: `TAG_NAME`

**Review by vraravam — CHANGES_REQUESTED** [review]

File: .github/workflows/docker.yml (hunk 2 above)
Comment (SpecialAro) [reply .github/workflows/docker.yml]: Thank you @vraravam 😄 Pushed a new commit to solve this. Can you take a look, please?

File: .github/workflows/docker.yml [hunk 3]
```diff
@@ -49,6 +49,7 @@ jobs:
             git push origin main --no-verify
             git tag -f $TAG_NAME
             git push origin --tags --no-verify
+            echo "TAG_NAME=${TAG_NAME:1}" >> $GITHUB_ENV
```
Comment (vraravam) [inline .github/workflows/docker.yml:52]: if we want to follow semver - we should default to `1.0.0`

**Review by vraravam — CHANGES_REQUESTED** [review]

File: .github/workflows/docker.yml (hunk 3 above)
Comment (SpecialAro) [reply .github/workflows/docker.yml:52]: We can, but I'm using raw a few lines bellow so we can keep `v1.0.0` on GH tags and `1.0.0` on dockerhub tags.

**Review by vraravam — APPROVED** [review]

**Pull request merged**